examples/mpsc-crypto-mining/src/main.rs

```diff
@@ -0,0 +1,76 @@
+#![feature(iterator_step_by)]
```

Comment:
I didn't realize this was here when you first linked it. We should try to avoid examples that use unstable features

Reply:
Removed. As it is no longer unstable.